src/main/convert.js

```diff
@@ -110,16 +110,26 @@ export class ThriftFileConverter {
 
   generateEnumKeys = (def: Enum) => `${def.definitions.map(d => `"${d.id.name}"`).join(' | ')};`;
 
+  generateEnumMap = (def: Enum) => {
+    const header = `{`;
+    const values = def.definitions.map((d, index) => `  "${d.id.name}": ${d.value ? d.value.value : index},`).join('\n');
+    const footer = `}`;
+
+    return [header,values,footer].join('\n');
+  };
+
   generateEnum = (def: Enum) => {
     if (this.enumvalues) {
       return `export type ${this.transformName(def.id.name)} = ${this.generateEnumValues(def)};
-       export type ${this.transformName(def.id.name)}Keys = ${this.generateEnumKeys(def)};`;
+       export type ${this.transformName(def.id.name)}Keys = ${this.generateEnumKeys(def)};
+       export const ${this.transformName(def.id.name)}Map = ${this.generateEnumMap(def)};`;
     }
 
     return `export type ${this.transformName(def.id.name)}Values = ${this.generateEnumValues(
       def
     )};
-      export type ${this.transformName(def.id.name)} = ${this.generateEnumKeys(def)};`;
+      export type ${this.transformName(def.id.name)} = ${this.generateEnumKeys(def)};
+      export const ${this.transformName(def.id.name)}Map = ${this.generateEnumMap(def)};`;
```

Comment:
Hmmm, this kinda defeats the purpose of the type name suffix, which will lead to linting errors. Actually it looks like the previous change which added `xxxKeys` broke it too, but I didn't notice. Easy solution is to move the `Map` and `Keys` suffix into the call to `transformName`